packages/theme-cart/theme-cart.js

```diff
@@ -83,6 +83,24 @@ export function addItem(id, options) {
   return request.cartAdd(id, options.quantity, options.properties);
 }
 
+/**
+ * Add a new line item to the cart from a product form
+ * @param {object} form The product form element
+ * @returns {Promise} Resolves with the line item object (See response of cart/add.js https://help.shopify.com/en/themes/development/getting-started/using-ajax-api#add-to-cart)
+ */
+export function addItemFromForm(form) {
+  validate.form(form);
+
+  var formData = new FormData(form);
+  if (!formData.get('quantity')) {
+    formData.append('quantity', 1);
+  }
```

Comment:
Is this necessary? 
We don't need to explicitly pass a quantity with the form (I tested it). It will default to 1 if there is no quantity element, which is what I expect based on the API's description.

Reply:
I just realized we don't need a quantity nor it needs to be valid. The API will default to 1 if it's NaN or empty. Thanks, removed it.

_(Btw, can you link me to this API page where it confirms that it's not required? I couldn't find it)_

Reply:
Awesome. Thanks :). 
I'll try to find it. I read it a _long_ time ago, but not sure it's still there - all the docs look different now